.github/workflows/Lib edFi.ods.standard.yml

```diff
@@ -34,7 +34,6 @@ jobs:
     with:
       calling_branch: ${{ github.head_ref || github.ref_name }}
   build:
-    if: ${{ always() }}
```

Comment:
Is there a reason to use `if: ${{ always() }}`?
I'm removing it since it [makes the workflows un-cancellable](https://docs.github.com/en/actions/learn-github-actions/expressions#always) and interferes with the `concurrency` setting.